### DISCUSSION TITLE
Notifying Sunbird 4.10 Users of Updated Terms and Policies

### DISCUSSION THREAD

**Discussion by vishalchincholi1** [Sunbird-Ed/Community#734] [General Discussion]
Hi there, We utilize Sunbird version 4.10; in this scenario, is there a method to notify current users of new terms and conditions? 
What would happen to current users in the Sunbird Registry if the Terms and Policies statement were to change is what we would like to know. 

Are there any documents on the subject that we might consult, and is there anyone on the Sunbird team who could assist us with this?
@Krishnaj20 @rajeevsathish 
![image](https://github.com/Sunbird-Ed/Community/assets/124983642/8303af22-5acb-42aa-b044-4a04f7618f77)

**Comment by HarishGangula** [comment]
@vishalchincholi1  These terms and conditions are stored in blob as html file with config set to that path, The config exists as part of lern API `/data/v1/system/settings` with `tncConfig`

**Comment by HarishGangula** [comment]
As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
